Makefile

```diff
@@ -133,6 +153,17 @@ $(FLAGDIR)/%.flags: %/Dockerfile $(FLAG_SH)
 	$(SHELL) $(FLAG_SH) $< >$@
 
 #
+# Generate .test.rd files, adding a test target for an image to the dependants
+# of its corresponding image.
+#
+# Note that the .test.rd files need to be updated if the Makefile changes
+# because the Makefile is what generates the .test.rd files.
+#
+$(TEST_RDEPS): depends/%.test.rd: Makefile
+	-mkdir -p $(dir $@)
+	echo $*.dependants: test-$* >"$@"
```

Comment:
Yup. Good catch. The TEST_RDEPS ones I can do in the Makefile; the image ones I'll have to do in the .d files.